content/docs/faq/README.md

```diff
@@ -150,7 +150,16 @@ spec:
 
 Default `duration` is [90 days](https://github.com/cert-manager/cert-manager/blob/v1.2.0/pkg/apis/certmanager/v1/const.go#L26). If `renewBefore` has not been set, `Certificate` will be renewed 2/3 through its _actual_ duration.
 
-### Why do you say that passwords on JKS or PKCS#12 files aren't helpful?
+<div id="keystore-passwords"></div>
+### Why are passwords on JKS or PKCS#12 files not helpful?
+
+This question comes in many forms, including:
+
+- "Why is it OK to hard code these passwords?"
+- "Do I need to keep these passwords secure?"
+- "Are these passwords used in a secure way?"
```

Comment:
I am not sure if I understand this question? WDYM with "secure way"?

Reply:
I meant like "If I use one of these passwords, is it handled like private data?" - to which we'd answer "you shouldn't care, because they're not helpful" 😂

Reply:
Should I change it?